CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -128,9 +128,13 @@ namespace l1t {
       // veto region is seed tower +/- <=egIsoVetoNrTowersPhi
       unsigned isoVetoNrTowersPhi_;
 
+      // turn on/off EG ID cuts
```

Comment:
Hi @rekovic has this change in conditions scheme been discussed with @ggovi ?

Anyway failures in the pr tests are related to this change without a corresponding gt change ( but nominally conditions scheme evolution is not allowed)